src/HttpClient/HttpClient.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\HttpClient;
+
+use Tempest\Container\InitializedBy;
+use Tempest\Http\Response;
+
+#[InitializedBy(HttpClientInitializer::class)]
+interface HttpClient
```

Comment:
Shouldn't we add a `send(Request $request)` as well?

Reply:
Should we? I'm happy to, just not sure how often that'd get used.

Reply:
Difficult one 😅

It might be overkill. OTOH, if we decide to add it later, it'll be a breaking change :/ But we might never want to add it.

Reply:
Did some more thinking on it, I actually think it'll be very useful in cases where you're integrating with APIs that don't have proper PHP SDKs. You could make a base request class or interface + trait combo that sets up the right headers etc, and then use that for all subsequent requests.

Reply:
> You could make a base request class or interface + trait combo that sets up the right headers etc, and then use that for all subsequent requests.


I'd be incapable of doing this, actually. Some other developer might insult PSR in such a manner, however. 😛 

I'll add it.